assets/queries/terraform/aws/autoscaling_groups_supply_tags/query.rego

```diff
@@ -16,3 +16,19 @@ CxPolicy[result] {
 		"searchLine": common_lib.build_search_line(["resource", "aws_autoscaling_group", name], []),
 	}
 }
+
+CxPolicy[result] {
+	module := input.document[i].module[name]
+	keyToCheck := common_lib.get_module_equivalent_key("aws", module.source, "aws_autoscaling_group", "tags")
+
+	not common_lib.valid_key(module, keyToCheck)
+
```

Comment:
I think here we could add another check to [`tags_as_map`](https://registry.terraform.io/modules/terraform-aws-modules/autoscaling/aws/latest#input_tags_as_map), since it is also accepted as tag too